### PR TITLE
Added Relay article

### PR DESCRIPTION
**Pull request by chenkie**
Still need to update the app with a few things. Also, still need to put in the TL;DR and the Aside showing auth.

screencap

### CODE REVIEW

File: _posts/2015-10-04-getting-started-with-relay.markdown [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+
+Those familiar with Flux will know that it allows apps to have one-way data flow which means that the data needs for each component are easy to reason about. Before Relay, however, Flux didn't provide a good way to interact with the server. Relay can be thought of as the next evolution of Flux.
+
+Crafting application data sources can often come down to a series of tradeoffs between  performance and duplication of code. If we wanted to make sure all of the data our application needs is ready to go at the same time, we could set up an endpoint to request everything. However, this will obviously have implications for performance. We could of course define a lot of custom endpoints that serve us only the data we need, but this means that we'll have to write a lot of code and potentially make many HTTP requests. In either case, if we were to ever make a change to our components that would have them require more data, we would probably need to adjust the structure of the data coming from our server. Ultimately, this arrangement between client and server can be somewhat brittle.
```
Comment (mgonto) [inline _posts/2015-10-04-getting-started-with-relay.markdown]: two spaces

File: _posts/2015-10-04-getting-started-with-relay.markdown [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+
+To address these issues, Relay allows us to specify exactly the shape of the data that we want from the server for a given component. Since we can place our queries within our components, we get the benefit of being better able to reason about that data associations for each component.
```
Comment (mgonto) [inline _posts/2015-10-04-getting-started-with-relay.markdown]: Add a digram after this showing what Relay is. Similar to the one we did with Falcor vs regular APIs

File: _posts/2015-10-04-getting-started-with-relay.markdown [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+
+## Getting Started
+
+In this tutorial we will take a cursory look at Relay and GraphQL by building a simple application that allows us to select a user and display which programming conferences they are registered to attend. This tutorial will focus on the core concepts of Relay and will only cover retrieving data. We will, however, see how we can write data in the next article of this series.
+
+![relay react](https://cdn.auth0.com/blog/relay/relay-1.png)
+
+### Schemas
+
+Since Relay uses GraphQL, we need to expose a GraphQL server that serves data when requested. Aside from setting up the database interaction, the first step in setting up a GraphQL server is to define a schema for it to use. The schema can be thought of as the place where we register anything that we want to pull out of GraphQL. This includes declarations for our data types, queries, and mutations (changes to data). GraphQL enforces strong typing and provides its own methods for us to register types.
```
Comment (mgonto) [inline _posts/2015-10-04-getting-started-with-relay.markdown]: Can't we show the schema code right in here? There's a lot fo text until the code

File: _posts/2015-10-04-getting-started-with-relay.markdown [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+    });
+  }
+  render() {
+    return(
+      <div className="container">
+        <select className="form-control" onChange={value => this.changeUser(value) }>
+          <option value="1">Ryan</option>
+          <option value="2">George</option>
+        </select>
+        <h2>{this.props.user.name} Conferences</h2>
```
Comment (mgonto) [inline _posts/2015-10-04-getting-started-with-relay.markdown]: What's this edges?

Comment (mgonto) [inline _posts/2015-10-04-getting-started-with-relay.markdown]: Where does it come from?

**Comment by chenkie** [comment]
SEO checklist: https://docs.google.com/a/auth10.com/document/d/1Tin2EP68oTyEBjLam-jFav9_FhA00iS_GhTa_exB5Rg/edit?usp=sharing

**Pull request merged**